From 7e4ed5386dc1e8f6c452ef8b58a2750051f437af Mon Sep 17 00:00:00 2001
From: Summer Qin <summer.qin@arm.com>
Date: Wed, 28 Jul 2021 11:19:12 +0800
Subject: [PATCH 4/4] Align with mbedtls-3.0.0

PSA_ALG_MD4 is removed from mbedtls-3.0.0. Use PSA_ALG_MD5 instead.

Signed-off-by: Summer Qin <summer.qin@arm.com>
---
 .../common/nspe/initial_attestation/pal_attestation_crypto.c    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/api-tests/platform/targets/common/nspe/initial_attestation/pal_attestation_crypto.c b/api-tests/platform/targets/common/nspe/initial_attestation/pal_attestation_crypto.c
index 04d9f42..daf7842 100644
--- a/api-tests/platform/targets/common/nspe/initial_attestation/pal_attestation_crypto.c
+++ b/api-tests/platform/targets/common/nspe/initial_attestation/pal_attestation_crypto.c
@@ -41,7 +41,7 @@ static psa_algorithm_t cose_hash_alg_id_to_psa(int32_t cose_hash_alg_id)
         status = PSA_ALG_SHA_256;
         break;
     default:
-        status = PSA_ALG_MD4;
+        status = PSA_ALG_MD5;
         break;
     }
 
-- 
2.17.1

